wgs/zk/README.md

```diff
@@ -8,3 +8,11 @@
 | 2 | Feb 7, 2024 | [Agenda Link](https://docs.google.com/presentation/d/1jTf28upZBGwrrD8DiFVWTZZtNj_5XvNHhAtqIJuQWtY/edit?usp=drive_link) | Notes Link | [Recording Link](https://drive.google.com/file/d/1v9iB_mNRp9_oIZWMX6PFO652y0jnBvHD/view?usp=drive_link) |
 | 3 | Feb 21, 2024 | Agenda Link | Notes Link | [Recording Link](https://drive.google.com/file/d/1tjjbiXfgbbWcRHuo1RH51OvmLKIJPIhQ/view?usp=drive_link) |
 | 4 | March 6, 2024 | Agenda Link | Notes Link | [Recording Link](https://drive.google.com/file/d/1iTa6U6BSsLMcaHJ4rB-KlLPOJInlJ7r2/view?usp=drive_link) |
+| 4 | March 21, 2024 | Agenda Link | Notes Link | [Recording Link](https://drive.google.com/file/d/1vKg8qA6tSAXPopt6G0Bsgf9jdAAjeN3u/view?usp=drive_link) |
```

Comment:
```suggestion
| 5 | March 21, 2024 | Agenda Link | Notes Link | [Recording Link](https://drive.google.com/file/d/1vKg8qA6tSAXPopt6G0Bsgf9jdAAjeN3u/view?usp=drive_link) |
```

Reply:
Should this video be made public?

<img width="677" alt="Screenshot 2024-04-01 at 10 14 29 AM" src="https://github.com/celestiaorg/CIPs/assets/3699047/9f0c144a-521b-4218-80dd-b60278525608">

Reply:
Fixed 


Thank you for checking